src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -4414,6 +4414,207 @@ class StubGenerator: public StubCodeGenerator {
     return (address) start;
   }
 
+#ifdef COMPILER2
+
+static const int64_t bits2 = right_n_bits(2);
+
+  // In sun.security.util.math.intpoly.IntegerPolynomial1305, integers
+  // are represented as long[5], with BITS_PER_LIMB = 26.
+  // Pack five 26-bit limbs into three 64-bit registers.
+  void pack_26(Register dest0, Register dest1, Register dest2, Register src, Register tmp1, Register tmp2) {
```

Comment:
If there only use is in `poly1305` implementation, could you please prefix this methid with `poly1305_`?
```suggestion
  void poly1305_pack_26(Register dest0, Register dest1, Register dest2, Register src, Register tmp1, Register tmp2) {
```

Reply:
All done.